samples/matter/light_switch/README.rst

```diff
@@ -117,6 +117,24 @@ Matter light switch custom configurations
     :start-after: matter_light_bulb_sample_configuration_file_types_start
     :end-before: matter_light_bulb_sample_configuration_file_types_end
 
+.. _matter_light_switch_snippets:
+
+Snippets
+========
+
+.. |snippet| replace:: :makevar:`light_switch_SNIPPET`
+
+.. include:: /includes/sample_snippets.txt
+
+The following snippet is available:
+
+* ``lit_icd`` - Enables experimental LIT ICD support.
+
+  .. note::
+    This snippet requires custom ZAP configuration, located in :file:`snippets/lit_icd` directory.
+    When using this snippet, remember to configure and generate the correct ZAP files.
+    To do it run the ```west zap-generate``` and choose the :file:`snippets/lit_icd/light_switch.zap` file.
```

Comment:
But this is already done and the proper files are pulled in by the snippet itself, right? So it would be valid only in case someone wants to modify the data model in a custom way.

Reply:
yes, I think I can modify it this way:
```suggestion
    When using this snippet, remember to configure and generate the correct ZAP files if custom data model is needed.
    To do it run the ```west zap-generate``` and choose the :file:`snippets/lit_icd/light_switch.zap` file.
```